gwt-material/src/test/java/gwt/material/design/client/ui/MaterialModalTest.java

```diff
@@ -41,21 +41,6 @@ public void init() {
         checkType(modal);
         checkDimissible(modal);
         checkDuration(modal);
-        checkMultipleModalZIndexes();
-    }
-
-    protected void checkMultipleModalZIndexes() {
```

Comment:
Might be good if we have a replacement for a test case on this part. We can make use of JSInterop to get each element z-indexes.